scripts/tsconfig.json

```diff
@@ -45,7 +45,7 @@
 
     /* JavaScript Support */
     "allowJs": true /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */,
-    "checkJs": true /* Enable error reporting in type-checked JavaScript files. */,
+    "checkJs": false /* Enable error reporting in type-checked JavaScript files. */,
```

Comment:
why change this?

Reply:
I don't think it's necessary for JS files...

Reply:
Actually, we didn't do type checking on the CI anyway and it make ts server complaining. So I turned it off.

Reply:
does not it provide autocomplete

Reply:
Yes, it does ;--)